src/email_utils.py

```diff
@@ -36,6 +55,21 @@ def send_email(logger: Logger, subject: str, msg: str, recipient: str | None = N
         logger.error(f"email '{subject}': {ex}")
     return False
 
+def send_email_postfix(logger: Logger, subject: str, msg: str, recipient: str | None = None, attachment: str | None = None) -> bool:
+    if not recipient:
+        recipient = app.config["ADMIN_EMAIL"]
+    from_email = app.config["FROM_EMAIL"]
+    html = render_template('email.html', content=msg)
+    message = Message(sender=from_email, recipients=[recipient], subject=subject, body=html)
+    if attachment:
+        message.attachments = attachment
+    try:
+        mail.send(message)
+        return True
+    except Exception as ex:
+        logger.error(f"email '{subject}': {ex}")
+    return False
+
```

Comment:
```suggestion
def send_email_postfix(logger: Logger, from_email: str, subject: str, msg: str, recipient: str, attachment: str | None = None) -> bool:
    html = render_template('email.html', content=msg)
    message = Message(sender=from_email, recipients=[recipient], subject=subject, body=html)
    if attachment:
        message.attachments = attachment
    try:
        mail.send(message)
        return True
    except Exception as ex:
        logger.error(f"email '{subject}': {ex}")
    return False
```

also make the `send_email_sendgrid` function similar

Reply:
edited this part as close to send_email_sendgrid and send_mail_postfix